packages/titus-backend/Dockerfile

```diff
@@ -1,15 +1,11 @@
-FROM node:16-alpine as builder
-RUN apk add dumb-init
+FROM node:16.13.1-alpine3.14 as builder
 ENV NODE_ENV production
-
 WORKDIR /service
-
-COPY --chown=node:node package.json package-lock.json ./
-COPY --chown=node:node lib ./lib
-COPY --chown=node:node index.js ./index.js
-
-RUN npm ci --only=production
-
+COPY --chown=node:node package.json package-lock.json index.js ./
+COPY --chown=node:node lib lib
+RUN apk upgrade --available && \
+    apk add dumb-init && \
+    rm -rf /var/cache/apk/* && \
+    npm ci --only=production
```

Comment:
can we keep unrelated commands in different run instructions please? so we don't mix them up

Reply:
Keeping image layers to a minimum is one of Docker's best practices as a way of reducing their size and improving their handling. As a drawback, I agree in that it goes against readability. It's just a matter of choosing one way or the other. What do you prefer?

Reply:
agreed, nonetheless I think we can accept the compromise to slightly increase readability here, but I don't have a strong opinion either

Reply:
OK, let's go for the readability path then.